package.json

```diff
@@ -68,12 +68,12 @@
     "fibers": "^2.0.0",
     "flatten-obj": "^3.1.1",
     "font-awesome": "^4.7.0",
-    "graphql": "0.13.1",
```

Comment:
So we went from version 0 to version 14?

Reply:
Yes, that's right. I was surprised too. `0.13.1` is nine months ago